tools/ossf_best_practices/silver_criteria.md

```diff
@@ -0,0 +1,321 @@
+Check the official [report](https://bestpractices.coreinfrastructure.org/en/projects/29?criteria_level=1) as some questions include additional information that might be relevant to understand the context around the question.
+
+
+# Basics
+> What is the human-readable name of the project?
+
+Node.js
+
+> What is a brief description of the project?
+
+Node.js® is a JavaScript runtime built on Chrome's V8 JavaScript engine
+
+> What is the URL for the project (as a whole)?
+
+https://nodejs.org
+
+> What is the URL for the version control repository (it may be the same as the project URL)?
+
+https://github.com/nodejs/node
+
+## Prerequisites
+
+> The project MUST achieve a passing level badge.
+
+**Met**
+
+
+## Basic project website content
+
+> The information on how to contribute MUST include the requirements for acceptable contributions (e.g., a reference to any required coding standard). (URL required) 
+
+**Met**
+https://github.com/nodejs/node/blob/master/CONTRIBUTING.md
+
+## Project oversight
+
+> The project SHOULD have a legal mechanism where all developers of non-trivial amounts of project software assert that they are legally authorized to make these contributions. The most common and easily-implemented approach for doing this is by using a [Developer Certificate of Origin (DCO)](https://developercertificate.org/), where users add "signed-off-by" in their commits and the project links to the DCO website. However, this MAY be implemented as a Contributor License Agreement (CLA), or other legal mechanism. (URL required)
+
+_Possible answers: Met/Unmet_
+
+> The project MUST clearly define and document its project governance model (the way it makes decisions, including key roles). (URL required)
+
+_Possible answers: Met/Unmet_
+
+> The project MUST adopt a code of conduct and post it in a standard location.
+
+_Possible answers: Met/Unmet_
+
+> The project MUST clearly define and publicly document the key roles in the project and their responsibilities, including any tasks those roles must perform. It MUST be clear who has which role(s), though this might not be documented in the same way. (URL required)
+
+_Possible answers: Met/Unmet_
```

Comment:
Maybe there are other documents that are more detailed, but the Governance is enough 

```suggestion
> The project MUST clearly define and publicly document the key roles in the project and their responsibilities, including any tasks those roles must perform. It MUST be clear who has which role(s), though this might not be documented in the same way. (URL required)

**Met**
https://github.com/nodejs/node/blob/main/GOVERNANCE.md
```